subgraphs/abracadabra/src/common/getters.ts

```diff
@@ -201,7 +206,7 @@ export function getOrCreateLendingProtocol(): LendingProtocol {
   LendingProtocolEntity = new LendingProtocol(getBentoBoxAddress(dataSource.network()));
   LendingProtocolEntity.name = "Abracadabra Money";
   LendingProtocolEntity.slug = "abracadabra";
-  LendingProtocolEntity.schemaVersion = "1.2.0";
+  LendingProtocolEntity.schemaVersion = "1.2.1";
   LendingProtocolEntity.subgraphVersion = "1.0.0";
```

Comment:
Should also update the subgraph version for every PR

Reply:
gotcha